src/coreclr/tools/aot/crossgen2/Program.cs

```diff
@@ -229,13 +229,11 @@ private InstructionSetSupport ConfigureInstructionSetSupport()
             // Ready to run images are built with certain instruction set baselines
             if ((_targetArchitecture == TargetArchitecture.X86) || (_targetArchitecture == TargetArchitecture.X64))
             {
-                instructionSetSupportBuilder.AddSupportedInstructionSet("sse");
-                instructionSetSupportBuilder.AddSupportedInstructionSet("sse2");
+                instructionSetSupportBuilder.AddSupportedInstructionSet("sse2"); // Lower baselines included by implication
```

Comment:
Can't we use AVX2 by default?

Reply:
That is https://github.com/dotnet/designs/pull/173